data/base/shaders/skybox.vert

```diff
@@ -0,0 +1,41 @@
+// Version directive is set by Warzone when loading the shader
+// (This shader supports GLSL 1.20 - 1.50 core.)
+
+uniform mat4 posMatrix;
+
+uniform int fog_enabled;
+uniform vec4 fog_color;
+
+#if (!defined(GL_ES) && (__VERSION__ >= 130)) || (defined(GL_ES) && (__VERSION__ >= 300))
+in vec4 vertex;
+in vec2 vertexTexCoord;
+in vec4 vertexColor;
+#else
+attribute vec4 vertex;
+attribute vec2 vertexTexCoord;
+attribute vec4 vertexColor;
+#endif
+
+#if (!defined(GL_ES) && (__VERSION__ >= 130)) || (defined(GL_ES) && (__VERSION__ >= 300))
+out vec2 uv;
+out vec4 vColour;
+out vec4 fog;
+#else
+varying vec2 uv;
+varying vec4 vColour;
```

Comment:
Also, _color_ vs _colour_ (US/GB).

Reply:
Yeah ik this problem is all over the codebase. We should fix it in a separate PR.